release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -557,6 +566,11 @@ In the table below, features are marked with the following statuses:
 |TP
 |
 
+|OpenShift sandboxed containers
```

Comment:
There's not really a rhyme or reason to where things are added to this table, so if you want you could just stick it at the end since it's new. Here is fine too, though :+1:

Reply:
I'll leave it here for now, unless someone requests it to move it :-)